core/templates/pages/contributor-dashboard-admin-page/contributor-admin-dashboard-page.component.html

```diff
@@ -1,67 +1,184 @@
+              Question<br>Reviewers
+            </a>
+          </li>
+        </ul>
+      </div>
+      <div class="filter-container">
+        <div class="language-selector-container" *ngIf="(activeTab === TAB_NAME_TRANSLATION_SUBMITTER) || (activeTab === TAB_NAME_TRANSLATION_REVIEWER)">
+          <div #languageDropdown class="oppia-filter-selector-container e2e-test-language-selector"
+               (click)="toggleLanguageDropdown()">
+            <div class="oppia-filter-selector-inner-container e2e-test-language-selector-selected">
+              <b>Language: </b>{{selectedLanguage}}
+              <span class="fas fa-caret-down oppia-filter-selector-inner-container-arrow"
+                    [@languageDropdown]="languageDropdownShown ? 'expanded' : 'collapsed'">
+              </span>
+            </div>
+            <div *ngIf="languageDropdownShown"
+                 class="oppia-filter-selector-dropdown-container e2e-test-language-selector-dropdown">
+              <div class="oppia-filter-selector-dropdown-option e2e-test-language-selector-option"
+                   *ngFor="let option of languages"
+                   (click)="selectLanguage(option.language)"
+                   [ngClass]="{'oppia-filter-selector-dropdown-option-selected' : option === selectedLanguage}">
+                {{option.language}}
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="filter-sub-container">
+          <div class="last-activity-selector-container">
+            <div #activityDropdown  class="oppia-filter-selector-container e2e-test-last-activity-selector"
+                 (click)="toggleActivityDropdown()">
+              <div class="oppia-filter-selector-inner-container e2e-test-last-activity-selector-selected">
+                <b>Filter by last activity: </b>
+                   {{ selectedLastActivity ? 'Within past ' + selectedLastActivity + ' days' : 'All time' }}
+                <span class="fas fa-caret-down oppia-filter-selector-inner-container-arrow"
+                      [@lastActivityDropdown]="activityDropdownShown ? 'expanded' : 'collapsed'">
+                </span>
+              </div>
+              <div *ngIf="activityDropdownShown"
+                   class="oppia-filter-selector-dropdown-container e2e-test-last-activity-selector-dropdown">
+                <div class="oppia-filter-selector-dropdown-option e2e-test-last-activity-selector-option"
+                     *ngFor="let option of lastActivty"
+                     (click)="selectLastActivity(option)"
+                     [ngClass]="{'oppia-filter-selector-dropdown-option-selected' : option === selectedLastActivity}">
+                  {{ option ? 'Within past ' + option + ' days' : 'All time' }}
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="topic-selector-container">
+            <oppia-topic-filter
```

Comment:
From the screencast, the topic filter is shorter than the last activity filter next to it. Can we make them the same height to be consistent?

Reply:
The topic filters height changes according to the number of topic selected, so it's not fixed but dynamic. 

Hence I have kept the height to change according to the content in it. Does that sounds reasonable?

Reply:
Well at the very least I think the heights should be the same in the defaults states—otherwise it looks strange.

Reply:
I tried fixing it but since both uses different native elements and one's height being dynamically changing I am not able to make it of same height.

Reply:
I have changed the video in description of the PR. PTAL. Does that looks good?

Reply:
Hm the filter looks the same to me?

Reply:
Sorry the video didn't get updated last time. PTAL again.

Reply:
Yeah I have updated it. The topic filter is slightly above but it's the same height now.

Reply:
@sagangwee Does the topic filter look good for you?